src/core/biblio.js

```diff
@@ -93,24 +93,76 @@ export function wireReference(rawRef, target = "_blank") {
   `;
 }
 
-export function stringifyReference(ref) {
+// Author, A. (Year, Month Date of Publication). Article title. Retrieved from URL
+function entryToAPA(ref) {
```

Comment:
So, based on @saschanaz's feedback, we should just be treating these as HTML objects, and not text... so we need refactor these like so:

```JS
const html = (...args) => hyperHTML.bind(doc.createDocumentFragment())(...args)

function entryToAPA(ref) {
  const { title } = ref;
  const authors =
    ref.authors && ref.authors.length
      ? `${ref.authors.join("; ")} ${ref.etAl ? " et al" : ""}.`
      : "";
  const date = ref.date ? `(${ref.date})` : "";
  return html`
    ${authors} ${date} <a href="${ref.href}"><cite>${title}</cite></a>
    ${ref.href ? html`Retrieved from URL: ${html`<a href="${ref.href}">${ref.href}</a>`}` : ""}
  `;
}

function entryToMLA(ref) {
   // as above, but for MLA
}

function entryToW3C(ref) {
  // as above, but for W3C...
}
```

The we can just add them directly to the document as real HTML elements.